### ISSUE TITLE
Hy client not showing output

### ISSUE THREAD

**Issue by madhat2r** [Olical/conjure#568]
I tried executing something very simple '(+1 2)' it appears that it does not execute at all.
```
; Sponsored by @campbellr ❤
 ; --------------------------------------------------------------------------------
 ; "hy --repl-output-fn=hy.repr" (started)
 ; --------------------------------------------------------------------------------
 ; eval (current-form): (+ 1 2)
```

I can start a repl in terminal using same command and it works:

```
Hy 0.28.0 using CPython(main) 3.11.4 on Linux
=> (+ 1 2)
3
=> 
```

Do you have any idea why this is not working?

**Comment by Olical** [comment]
Looking into this now, I suspect it's due to a change in Hy but I'm not sure what yet. I'm reading through the changelog and seeing if I can get it to work again.

**Comment by Olical** [comment]
https://github.com/hylang/hy/issues/2546 I think the existing code used the `ps1` override feature, so that might be part of it.

**Comment by Olical** [comment]
I think I fixed it on the `develop` branch? It's to do with recent changes to Hy and maaaaybe it's a Hy bug? I think the two required things are `-i` to force it to start a REPL even though we're not running it under a TTY and I seem to need to give it a piece of code to execute first before it'll drop into a REPL. I opted to do this with `-c "Ready!"`, so it' evaluates that string THEN drops into a REPL.

Both options seem to be required now, I think really it should drop into a REPL with just `-i` and no `-c` but this works for us. Hope that helps!
